admin/agentlocal/agentlocal.go

```diff
@@ -77,6 +77,8 @@ type Status struct {
 	Connected        bool          `json:"connected"`
 	ServerClockDrift time.Duration `json:"server_clock_drift,omitempty"`
 	ServerLatency    time.Duration `json:"server_latency,omitempty"`
+
+	UpConnectedTime float32 `json:"up_connected_time"`
```

Comment:
How about just `uptime` or `uptime_sec` (if it is in seconds)

Reply:
@ritbl uptime imples how long the agent has been up, right? This is not that metric

Reply:
It's not in percentage which is counted by formula `number_of_successfull_ping_requests / number_of_total_ping_requests * 100`

I decided to use this name to clarify that it's uptime of connection between agent and server. Otherwise it can be interpretated in some other ways, like uptime of agent for example.